lib/client/ticks.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+function prepare (client, opts) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (8)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1156649228/issues/source?bid=2452379&fileBranchId=2909529#l3)